resources/overlay_plugin_api.ts

```diff
@@ -54,14 +54,15 @@ type VoidFunc<T> = (...args: T[]) => void;
 
 let inited = false;
 
-let wsUrl: RegExpExecArray | null = null;
+let wsUrl: string | null = null;
 let ws: WebSocket | null = null;
 let queue: (
   | { [s: string]: unknown }
   | [{ [s: string]: unknown }, ((value: string | null) => unknown) | undefined]
 )[] | null = [];
 let rseqCounter = 0;
 const responsePromises: Record<number, (value: unknown) => void> = {};
+const urlParams = new URLSearchParams(window.location.search);
```

Comment:
`window` may not be defined if this is running from node.  (See the errors in post process.)  What about moving the `const urlParams` declaration down to where it is used on line 199?

Also, would you mind moving the `wsUrl` declaration down to line 199 as well? It could be `const wsUrl`.

Reply:
There's no rush, but just wanted to make sure this comment wasn't missed.  Do you have time to fix this issue?  You should be able to reproduce the error by running `npm run test` locally.

Reply:
A commit is pushed